helmfile.d/10-matrix.yaml

```diff
@@ -17,8 +17,20 @@ releases:
     macaroonSecretKey: {{ env "MACAROON_SECRET_KEY" | default "macaroon_secret" }}
     formSecret: {{ env "FORM_SECRET" | default "form_secret" }}
     signingKey: {{ env "SIGNING_KEY" | default "signing_key" | quote }}
+    antiscam: true
 
 - name: riot-web
   chart: ../charts/riot-web
   values:
   - environment: {{ .Environment.Name }}
+
```

Comment:
Same as above, this will only be taken into account on integration tests, maybe leave it for a followup. We need to add a block like this in the repos that depend on this chart and need to install it, like `community-matrix-server`.